logic_control/logic_control.c

```diff
@@ -28,11 +38,57 @@ bool funcb(){
 }
 
 void get_and_send_data(void){
+	//min max , data na mapovanie
+
+	//toto neviem co s tym chces robit
 	lr = map(*RC_Commands,806,4032,-100,100);
 	fb = map(*(RC_Commands+1),347,3400,-100,100);
 	ud = map(*(RC_Commands+2),3155,840,-100,100);
 	yv = map(*(RC_Commands+3),3117,467,-100,100);
 
+
+	if((map((*RC_Commands),(*RC_Commands),4032,0,100)<=(map((*RC_Commands),(*RC_Commands)+deadzone,4032,0,100)))&&(map((*RC_Commands),(*RC_Commands),829,0,-100)>=(map((*RC_Commands),(*RC_Commands)-deadzone,829,0,-100)))){
+		x_speed = 0;
+	}
+	if((map((*RC_Commands),(*RC_Commands),4032,0,100))>(map((*RC_Commands),(*RC_Commands)+deadzone,4032,0,100))){
+		x_speed = map((*RC_Commands),(*RC_Commands),4032,0,100);
+	}
+	if((map((*RC_Commands),(*RC_Commands),829,0,-100))<(map((*RC_Commands),(*RC_Commands)-deadzone,829,0,-100))){
```

Comment:
The same goes for this and all others statments, 
This should look like:
`if(xposition>reset_pos+deadband){ x_speed = map(xposition, reset_pos-deadband, high_x,0,100); }`
Try to follow standard KISS